### PR TITLE
added cloudbeat fix

### PR DESCRIPTION
**Pull request by DaveSys911**
Cloudbeat rename agent bugfixes

### CODE REVIEW

File: cloudbeat/cloudbeat.yml [hunk 1]
```diff
@@ -113,10 +114,9 @@ output.elasticsearch:
 # Configure processors to enhance or manipulate events generated by the beat.
 
 processors:
-#  - add_host_metadata: ~
-#  - add_cloud_metadata: ~
-#  - add_docker_metadata: ~
-  - add_cluster_id: ~
+  - add_host_metadata: ~
```
Comment (eyalkraft) [inline cloudbeat/cloudbeat.yml:117]: Bad change
1. `add_cluster_id` is a processor added by @oren-zohar and it's crucial and should be included here
2. regarding the other processors we can include them but i'm not sure we should... when running inside the agent, the agent takes care of all of these. so I wouldn't include them here.

**Review by eyalkraft — CHANGES_REQUESTED** [review]

File: cloudbeat/cloudbeat.yml (hunk 1 above)
Comment (DaveSys911) [reply cloudbeat/cloudbeat.yml:117]: This is auto-gen from the beat. What are the correct values here @oren-zohar ?

Comment (eyalkraft) [reply cloudbeat/cloudbeat.yml:117]: hmm ok just tagging @oren-zohar to make sure he'll know it

Comment (amirbenun) [reply cloudbeat/cloudbeat.yml:117]: That's a degradation, we need to add it back.
@DaveSys911 can you add `add_cluster_id: ~`?
If necessary let's create a ticket for @oren-zohar to add this processor properly.

**Review by amirbenun — CHANGES_REQUESTED** [review]

File: cloudbeat/cloudbeat.yml (hunk 1 above)
Comment (DaveSys911) [reply cloudbeat/cloudbeat.yml:117]: I can do it, but let's make it a separate pr.

**Review by amirbenun — APPROVED** [review]

**Review by eyalkraft — APPROVED** [review]

**Pull request merged**